intelmq/tests/bots/experts/trusted_introducer_lookup/test_data/teams.json

```diff
@@ -0,0 +1,554 @@
+[
```

Comment:
Can we shorten that file to a minimum required set for the tests? I assume 95% of this file is not used.